src/index.js

```diff
@@ -0,0 +1,99 @@
+import * as tk from './utility.js';
+import './style.css';
+
+const loadTask = document.querySelector('.todo-list');
+const form = document.querySelector('.todo-form');
+const clearList = document.querySelector('.clear');
+
+let taskIndex = 0;
+let taskDescription = '';
+const taskCompleted = '';
+
+const loadListElement = () => {
+  const li = document.createElement('li');
+  li.classList.add(
+    // 'list-group-item',
+    // 'justify-content-between',
+    // 'd-flex',
+    // 'align-items-center',
```

Comment:
- [ ] Kindly ensure to remove the commented code, it's not good practice to have commented code in your files.